lib/utils/get-role.js

```diff
@@ -0,0 +1,93 @@
+const {getProp, getPropValue} = require('jsx-ast-utils')
+const {elementRoles} = require('aria-query')
+const {getElementType} = require('./get-element-type')
+const ObjectMap = require('./object-map')
+
+// Clean-up `elementRoles` from `aria-query`
+const elementRolesMap = new ObjectMap()
```

Comment:
This code is constructing a cleaned up version of [elementRoles](https://github.com/A11yance/aria-query/blob/main/__tests__/src/elementRoleMap-test.js#L4). Below, we'll be constructing a key and grabbing the value from `elementRoles`.